data-access/fhir/config/default/extension-search-parameters.json

```
@@ -0,0 +1,9 @@
+{
+  "resourceType": "Bundle",
+  "id": "searchParams",
+  "meta": {
+    "lastUpdated": "2018-12-27T22:37:54.724+11:00"
```

What is the meaning of this date? Should it be updated to today, or is it specific to some point in time?

---

Its not really used for anything and could even be removed.
That said, maybe its useful to keep it as a way for users that change this file to note when it last changed...which I think is why it existed in this file to begin with (I just copied a placeholder one from our repo)
```suggestion
    "lastUpdated": "2021-05-06T07:20:00-04:00"
```

---

I'm also fine to remove it since the lastUpdated time should already be available for users familiar with git

---

decided to remove it (and the id)